lte/cloud/go/services/subscriberdb/config.go

```diff
@@ -22,7 +22,8 @@ import (
 
 type Config struct {
 	// FlatDigestEnabled is a feature flag for the flat digest functionality.
-	FlatDigestEnabled bool `yaml:"flatDigestEnabled"`
+	FlatDigestEnabled bool   `yaml:"flatDigestEnabled"`
+	SyncInterval      uint32 `yaml:"syncInterval"`
```

Comment:
Yeah add a docstring for this, so that readers don't have to refer to the .yml file